src/lib/token_list_gen.ts

```diff
@@ -219,7 +220,9 @@ export const generateTokenList = async (
         // parse null terminated bytes32 strings
         else if (_name.length === 64)
           return utils.parseBytes32String('0x' + _name);
-        else return _name;
+        else if (token.l2Address === BridgedUSDCContractAddressArb1) {
```

Comment:
should we perform `toLowerCase` on both sides?

Reply:
Good point